rtl/include/eclhelper.hpp

```diff
@@ -45,7 +45,7 @@ typedef unsigned short UChar;
 //Should be incremented whenever the virtuals in the context or a helper are changed, so
 //that a work unit can't be rerun.  Try as hard as possible to retain compatibility.
 #define ACTIVITY_INTERFACE_VERSION      165
-#define MIN_ACTIVITY_INTERFACE_VERSION  165             //minimum value that is compatible with current interface - without using selectInterface
+#define MIN_ACTIVITY_INTERFACE_VERSION  165             //minimum value that is compatible with current interface
```

Comment:
we should increase this number!

Reply:
You don't think this change is compatible?

Reply:
Fixed